sdk/metrics/testing-internal/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("otel.java-conventions")
+}
+
+description = "OpenTelemetry Metrics Testing (Internal)"
+otelJava.moduleName.set("io.opentelemetry.metrics.internal.testing")
```

Comment:
IMO the code in here isn't valuable enough to justify organizing into a new module. I would just just create copies of the utility functions in each module where its needed.

Reply:
Ok, I'll revert the changes to `HistogramCollectBenchmark` and move those two classes into `jmhTest` directory